pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/SerDe.scala

```diff
@@ -282,3 +282,9 @@ object SerDe {
     value.foreach(v => writeBytes(out, v))
   }
 }
+
+object SerializationFormats {
```

Comment:
I'd like to continue with String, because it's easy to be passed between Scala and R, or you will have some helpful function to convert the string into Java object.